packages/x-charts-pro/src/internals/plugins/useChartProExport/print.ts

```diff
@@ -0,0 +1,91 @@
+import ownerDocument from '@mui/utils/ownerDocument';
+import { ChartPrintExportOptions } from './useChartProExport.types';
+
+export function printChart(
+  element: HTMLElement | SVGElement,
+  { fileName }: ChartPrintExportOptions = {},
+) {
+  const printWindow = buildPrintWindow(fileName);
+  const doc = ownerDocument(element);
+
+  printWindow.onload = async () => {
+    const printDoc = printWindow.contentDocument!;
+    const elementClone = element!.cloneNode(true) as HTMLElement | SVGElement;
```

Comment:
Instead of `cloneNode`, I wonder if we could use `importNode`. It would allow us to skip setting the `printDoc.body.innerHTML`.